common/postgresql-ng/values.yaml

```diff
@@ -37,9 +39,25 @@ config:
 
   random_page_cost: 1.1
 
+# users to create
+users:
+  # NOTE: backup, metrics and postgres are reserved user names and adding them here might cause unexpected issues
+  # acme-user:
+    # Ensure the user has those grant lines run on each start.
+    # TO $USER with the username is added per line.
+    # %PGDATABASE% is replaced with the actual database name.
+    #grant: GRANT CONNECT,SELECT ON DATABASE "%PGDATABASE%"
+    # give out a general set of default permissions to let an app manage its own db
+    #manageAppDb: true
+    # Alter the table owner to this user. Make sure to only set this once!
+    #setOwnerOnTable: true
+
+createMetricsUser: true
```

Comment:
I don't see a reason to not create a pgmetrics user, so we could put this user as an example here to document how this works.
```suggestion
# users to create; by default a metrics user for postgres-exporter is created
users:
  # NOTE: backup and postgres are reserved user names and adding them here might cause unexpected issues
  metrics:
    # Ensure the user has those grant lines run on each start.
    # TO $USER with the username is added per line.
    # %PGDATABASE% is replaced with the actual database name.
    # For example, a read-only application user would need `GRANT CONNECT,SELECT ON DATABASE "%PGDATABASE%"`
    grant: "GRANT pg_monitor"
    # give out a general set of default permissions to let an app manage its own db
    manageAppDb: false
    # Alter the table owner to this user. Make sure to only set this once!
    setOwnerOnTable: false
```

Reply:
The metrics user needs quite special permissions and I don't want to proxy them through the values if not necessary.